presto-docs/src/main/sphinx/connector/teradata.rst

```diff
@@ -0,0 +1,53 @@
+===================
+Teradata  Connector
+===================
+
+Teradata provides a low latency high performing connector that
+supports high concurrency, and parallel processing between Teradata
+and Presto. The Teradata QueryGrid connector for Presto enables users
```

Comment:
Sure. I basically copied this from the Teradata data sheet. But I can change it to be more clear. What do you suggest?